### ISSUE TITLE
While auto reading content from dialog screen-reader doesn't tell user about the presence of close button.

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4596]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 4.1.1 Parsing

**Test Environment:**
- Test URL: https://clarity.design/documentation/modals
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 19.3.1

**Steps to reproduce:**
1. Open the test URL.
2. Move focus onto to the first Show Modal button present under Angular modals.
3. Activate the button by pressing the enter or space key and observe the spoken feedback.

**Expected result:**
When the dialog is opened dialog content should be auto read appropriately.

**Actual result:**
1.Title is announced 4 times (reported as a separate issue).
2. Close button is not announced by screen reader while auto reading the dialog content. If the user  tabs/arrow to the close button, the button is focused and label is announced.

**Reporter Email:** gkhadke@vmware.com

**Comment by gkhadK** [comment]
I looked at this again and I felt not announcing the close button reduces verbosity. As screen reader have alternative options like pressing escape key or cancel button, current behavior seems to be better than what's requested in the expected result. So I think no change is required here.

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
